win32ss/user/ntuser/defwnd.c

```diff
@@ -529,6 +529,19 @@ DefWndScreenshot(PWND pWnd)
     UserCloseClipboard();
 }
 
+/* Is Window Snap enabled? */
+BOOL IsWindowSnapEnabled(VOID)
+{
+    WCHAR szValue[2];
+    if (RegReadUserSetting(L"Control Panel\\Desktop", L"WindowArrangementActive",
+                           REG_SZ, szValue, sizeof(szValue)))
+    {
+        szValue[RTL_NUMBER_OF(szValue) - 1] = UNICODE_NULL; /* Avoid buffer overrun */
+        return _wtoi(szValue);
```

Comment:
```suggestion
        return (_wtoi(szValue) != 0);
```

Reply:
Adding the double exclamation point should work perfectly as well. I don't have preference between the two versions. The "!= 0" is more common though.

Reply:
Added paren and added `!= 0`.